zerver/lib/integrations.py

```diff
@@ -74,14 +84,30 @@ def __init__(self, name, client_name=None, logo=None, secondary_line_text=None,
         self.url = url
 
         if doc is None:
-            doc = self.DEFAULT_DOC_PATH.format(name=name)
+            try:
+                doc = self.DEFAULT_DOC_PATH.format(name=name, ext='md')
+                loader.get_template(doc)
+            except loader.TemplateDoesNotExist:
+                doc = self.DEFAULT_DOC_PATH.format(name=name, ext='html')
```

Comment:
Actually, I have a concern here -- since this runs at import time, is `loader.get_template()` expensive?  If so, we might want to do something cheaper.  

I did some quick testing and confirmed that this appears to add like 10ms to the import time for this module:
```
# Post-branch
In [2]: print time.time(); import zerver.lib.integrations; print time.time()
1491416444.57
1491416444.7

# Pre-branch
In [2]: print time.time(); import zerver.lib.integrations; print time.time()
1491416460.92
1491416460.98
```

Should we just use an `os.path.join()` check to see which file exists rather than `loader.get_template()`?